src/sections/LoginAuth0.js

```diff
@@ -49,16 +59,32 @@ class LoginAuth0 extends Component {
         // authResult.expiresIn - string with the access token's expiration time in seconds
         // authResult.idToken - ID token JWT containing user profile information
 
+        const { name, email, picture } = authResult.idTokenPayload;
+
         this.props
           .authenticate({
             variables: {
-              accessToken: authResult.accessToken
+              accessToken: authResult.accessToken,
+              name,
+              email,
+              picture
             }
           })
           .then(({ data }) => {
             const userInfo = data.authenticateUser;
+            this.setState({ isLoggingIn: false });
```

Comment:
I changed the order of this and the following line to fix this warning:
<img width="680" alt="screen shot 2018-07-19 at 6 29 37 pm" src="https://user-images.githubusercontent.com/5432102/42977010-108bef90-8b82-11e8-8049-5baede38c41d.png">